<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDialogConfig</class>
 <widget class="QDialog" name="TDialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>511</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>初始化配置</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>341</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_import">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>导入配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_export">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>应用配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_removeRow">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>删除频点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_appendRow">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>新增频点</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
